homeassistant/components/zwave_js/light.py

```diff
@@ -451,6 +453,79 @@ def _calculate_color_values(self) -> None:
             self._color_mode = ColorMode.RGBW
 
 
+class ZwaveBasicLight(ZWaveBaseEntity, LightEntity):
```

Comment:
Does it differ that much that it needs its own (duplicated) class ?
Not possible to create a Base model and inherit the sub types from that ?

Reply:
yeah it's possible and I thought of doing so but then forgot. Will look into it again

Reply:
turns out while `ZwaveLight` has a bunch of extra logic not needed here, because of the way it is written, we are still good!